.github/workflows/check_codegen.yml

```diff
@@ -60,7 +60,7 @@ jobs:
         run: poetry install -v --only main
       - name: Generate gRPC stubs
         run: |
-          poetry run grpc-generator --proto-basepath ${{ inputs.proto-basepath }} --proto-subpath ${{ inputs.proto-subpath }} --output-basepath ${{ inputs.output-basepath }} --output-format ${{ inputs.output-format }}
+          poetry run grpc-generator --proto-basepath ${{ inputs.proto-basepath }} --proto-subpath ${{ inputs.proto-subpath }} --proto-include-path '../../third_party/ni-apis/ni/grpcdevice/v1' --output-basepath ${{ inputs.output-basepath }} --output-format ${{ inputs.output-format }}
```

Comment:
Instead of hardcoding this additional include, consider creating an optional input parameter which specifies additional include paths. This way it shows up at the top and is not buried in the workflow.

Reply:
Do we want to include this additional include path for all packages, or make it conditional for specific packages. We could specify optional parameter in the package matrix.

Reply:
I think an optional parameter is better but the current approach seems to work too.